larastan.neon

```diff
@@ -81,6 +81,11 @@ services:
                 - secure_asset
         tags:
             - phpstan.rules.rule
+    -
+        class: Worksome\CodingStyle\PHPStan\Laravel\Migrations\RequireWithoutTimestampsRule
+        tags:
+            - phpstan.rules.rule
```

Comment:
This needs to be moved to the `rules` parameter. 👍🏻 Basically just add this under `DisallowEnvironmentCheckRule`:
```yaml
- Worksome\CodingStyle\PHPStan\Laravel\Migrations\RequireWithoutTimestampsRule
```

Reply:
Thanks, moved!